.travis.yml

```diff
@@ -2,22 +2,76 @@
 language: c++
 sudo: required
 
+services:
+  - docker
+
 os:
   - linux
```

Comment:
question: does this line run 14.04? if so do we still want to be running 14.04? I'm thinking it's more trouble than its worth. If we are getting rid of it we could remove any scripts used only by 14.04.

Reply:
Docker service is only available on linux (which is 14.04). I can remove the 14:04 scripts responsible for building TECA

Reply:
hmm. Sorry but I am confused about your reply. We are talking about the os section are we not?

Reply:
huh, I'm confused. I meant to inquire about the os section, you seemed to reply about the services section...